include/bbp/reports/records.hpp

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <reports/library/reportinglib.hpp>
+
+/**
+ * Provides a bridge between c-based programs in order to have access to the c++-based objects
+ */
+#ifndef __cplusplus
+#include <stddef.h>
+#endif
+#if defined(__cplusplus)
+#include <cstddef>
+extern "C" {
+#endif
+int records_clear();
+/**
+ * Add cell to existing or new report
+ */
+int records_add_report(const char* reportName,
```

Comment:
why `const char*` instead of `std::string`?

Reply:
same than previous comment...coreneuron c calls